modules/cfe_testcase/src/es_misc_test.c

```diff
@@ -68,8 +68,15 @@ void TestWriteToSysLog(void)
                "MIR (Manual Inspection Required) for CFE_ES_WriteToSysLog");
 }
 
+void TestProcessAsyncEvent(void)
+{
+    UtPrintf("Testing: CFE_ES_ProcessAsyncEvent");
+    CFE_ES_ProcessAsyncEvent();
```

Comment:
Wrap in UtAssert_VOIDCALL (depends on nasa/osal#1125) so it gets reported in the log.